icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/MeasureUnitTest.java

```diff
@@ -3085,27 +3094,6 @@ static void generateCXXConstants() {
                     measureUnitToTypeSubType.get(entry.getKey()));
         }
 
-        System.out.println("// Must be sorted by first value and then second value.");
-        System.out.println("static int32_t unitPerUnitToSingleUnit[][4] = {");
```

Comment:
Deleting this block implements the TODO comment wherein the output was manually removed:

// unitPerUnitToSingleUnit no longer in use! TODO: remove from code-generation code.